builtin/game/falling.lua

```diff
@@ -42,8 +42,7 @@ core.register_entity(":__builtin:falling_node", {
 
 	on_step = function(self, dtime)
 		 -- Set gravity
-		local acceleration = self.object:getacceleration()
-		if not vector.equals(acceleration, {x = 0, y = -10, z = 0}) then
+		if self.object:getacceleration().y ~= -10 then
```

Comment:
It's gravity acceleration, which is independent of horizontal stuff, so this is ok, but ..